src/OpenDebugAD7/MILaunchOptions.cs

```diff
@@ -581,6 +603,22 @@ private static string GetLLDBMIPath()
                     xmlLaunchOptions.Append("    </Environment>\n");
                 }
 
+                if (jsonLaunchOptions.SymbolLoadInfo != null)
+                {
+                    xmlLaunchOptions.Append("    <SymbolLoadInfo ");
+                    if (jsonLaunchOptions.SymbolLoadInfo.LoadAll.HasValue)
+                    {
+                        xmlLaunchOptions.Append("LoadAll = '" + (jsonLaunchOptions.SymbolLoadInfo.LoadAll.Value ? "true" : "false") + "' ");
+                    }
+
+                    if (!String.IsNullOrWhiteSpace(jsonLaunchOptions.SymbolLoadInfo.ExceptionList))
+                    {
+                        xmlLaunchOptions.Append("ExceptionList='" + XmlSingleQuotedAttributeEncode(jsonLaunchOptions.SymbolLoadInfo.ExceptionList) + "' ");
```

Comment:
What happens if you don't specify LoadAll but there is an exception list? Is the behavior that LoadAll is true or false?

Reply:
> List of filenames (wildcards allowed). Modifies behavior of LoadAll. If LoadAll is true then don't load symbols for libs that match any name in the list. Otherwise only load symbols for libs that match.